src/views/tournament/AllTournamentsView.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+
+import ApiCommunication from "../../services/apicommunication/ApiCommunication";
+
+class AllTournamentsView extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            id: "",
+            tournaments: [],
+            selectedTournament: {}
+        };
+    }
+
+    getAllTournaments() {
+        const body = "query {tournaments{name}}";
+        const vars = "{}";
+        ApiCommunication.graphQlCallPost(body, vars)
+            .then(response => this.state.tournaments = response.data.data.tournaments)
+            .catch(error => console.log(error));
```

Comment:
logging error is not master code

Reply:
![image](https://user-images.githubusercontent.com/13750002/69946349-c00a6c00-14eb-11ea-9251-fa2bf6aa6a3f.png)

Reply:
![image](https://user-images.githubusercontent.com/13750002/69946439-e5977580-14eb-11ea-9644-10b77fe8e4c2.png)

Reply:
🤔

Reply:
OK made it console.error

Reply:
Response on: "Console.log is not allowed, but is in the master, so it is allowed".
By this reasoning:
School shootings zijn verboden, maar gebeuren, dus ze zijn legaal.
I guess we both understand that this reasoning is wrong